### PR TITLE
More test functions has been added.

### PR DESCRIPTION
**Pull request by ahmdrz**
1. TestNotModifiedStatusCode
2. Default cache dir changed to ~/.config/<target>
3. Makefile change for installable make.

### CODE REVIEW

**Comment by ahmdrz** [comment]
Your right , I will change README and Makefile.
Thank you.

**Comment by ahmdrz** [comment]
Done

**Comment by ahmdrz** [comment]
Sorry for my mistake
I change the README using my mobile device and that was happend.

**Comment by ahmdrz** [comment]
@siadat
What is your idea about in-memory cache ?
May I create in-memory cache with expiration time and after expired store in file.
I want to store contents that only has less than specific bytes in memory.

File: mina_test.go [hunk 1]
```diff
@@ -50,6 +50,57 @@ func TestMina(t *testing.T) {
 	}
 }
 
+func TestNotModifiedStatusCode(t *testing.T) {
+	// Ignoring sync.Mutex for two requests that are not concurrent
```
Comment (siadat) [inline mina_test.go]: This comment is unnecessary, right?

File: README.md [hunk 2]
```diff
@@ -11,7 +11,7 @@ Mina is saves API server responses to disk and serves them with its own HTTP ser
 
 ## Install
 
-    go get -u github.com/sariina/mina
+`go get -u github.com/sariina/mina`
```
Comment (siadat) [inline README.md]: What's the reason for this change? :thinking:

File: README.md [hunk 3]
```diff
@@ -35,16 +35,16 @@ Your app will think that you are using Github API even when you are offline.
     mina --help
 
     Usage:
-      mina -addr=<addr> -target=<target> [-o=<dir>] [-H=<header>]...
-    
+    mina -addr=<addr> -target=<target> [-o=<dir>] [-H=<header>]...
```
Comment (siadat) [inline README.md]: ... and also these indent changes? :coffee:

File: main.go [hunk 4]
```diff
@@ -18,7 +19,7 @@ const usage string = `Usage:
 Options:
   -addr    address to listen to
   -target  target to route to
-  -H       custom header
+  -H       [optional] custom header
```
Comment (siadat) [inline main.go]: :+1:

File: main.go [hunk 5]
```diff
@@ -64,14 +65,18 @@ func main() {
...
+		}
+		*flagCacheDir = fmt.Sprintf("%s/.config/mina/%s", usr.HomeDir, targetURL.Host)
 	}
 	*flagCacheDir, err = filepath.Abs(*flagCacheDir)
 	if err != nil {
 		log.Fatalln(err)
 	}
 	if !isFileExist(*flagCacheDir) {
-		err = os.Mkdir(*flagCacheDir, 0755)
+		err = os.MkdirAll(*flagCacheDir, 0755)
```
Comment (siadat) [inline main.go]: :cocktail:

**Comment by siadat** [comment]
@ahmdrz in-memory cache sounds cool, let's discuss the in-memory cache in another pull request :+1:

File: README.md (hunk 2 above)
Comment (ahmdrz) [inline README.md]: As I said , I made changes with my cellphone and I couldn't use indents :cry:
Honestly , I don't know why indents not working ...

File: mina_test.go (hunk 1 above)
Comment (ahmdrz) [inline mina_test.go]: Yep , By the way I wrote this , Shall I remove it ?

**Comment by ahmdrz** [comment]
So first of all , I will fix this pull request , And after you merge , I will make new PR.

**Comment by siadat** [comment]
@ahmdrz sounds good :metal:

**Comment by siadat** [comment]
Thank you! :tada:
